src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -36,7 +36,7 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         public CSharpParseOptions(
-            LanguageVersion languageVersion = LanguageVersion.CSharp6,
+            LanguageVersion languageVersion = LanguageVersion.CSharp7,
```

Comment:
I asked @jaredpar this prior to staring.  He said to make it the default as we are targeting the future branch.